.ci/docker/build.sh

```diff
@@ -308,7 +308,7 @@ case "$image" in
     TRITON=yes
     ;;
   pytorch-linux-jammy-xpu-2024.0-py3)
-    ANACONDA_PYTHON_VERSION=3.8
```

Comment:
Should we update to 3.9 as lowest supported python version instead ?

Reply:
Done. Thanks @atalman